src/components/Icons/InactiveStatusIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import { SvgIcon, SvgIconProps } from '@mui/material';
+
+export const InactiveStatusIcon = (props: SvgIconProps) => (
+  <SvgIcon {...props} viewBox="0 0 24 24">
+    <path d="M19 13H5V11H19V13Z" />
```

Comment:
Is this line intentionally not centered vertically?

Reply:
> Is this line intentionally not centered vertically?

Not sure about intentions, but this is what I received from Amy: [Figma file](https://www.figma.com/file/IfEmcJeuNY6ucgjYctlKB7/FOR-DEVELOPMENT---Cord-Field?type=design&node-id=11383-66102&mode=design&t=gV2rWeeeOH0FUwwk-0)

Reply:
Amy responded saying it can be adjusted down so I'm going to look at the code to see how other icons are adjusted to follow same process.  Also she asked that I disable/gray out the text for the word "Inactive" so I need to make that change too.